scene/animation/animation_mixer.cpp

```diff
@@ -33,7 +33,10 @@
 
 #include "core/config/engine.h"
 #include "core/config/project_settings.h"
+#include "scene/2d/audio_stream_player_2d.h"
+#include "scene/3d/audio_stream_player_3d.h"
 #include "scene/animation/animation_player.h"
+#include "scene/audio/audio_stream_player.h"
```

Comment:
Those seem unused, and cause a compilation error:
https://github.com/godotengine/godot/actions/runs/9128230788/job/25100184935?pr=91382

Remember the `disable_3d=yes` SCons option which disables compilation of most 3D nodes. So including 3D nodes in non-3D ones needs using `#ifndef _3D_DISABLED` guards (but here it doesn't seem used in the end, which is better to ensure encapsulation).

Reply:
This is still relevant